whisk.example.yaml

```diff
@@ -70,6 +74,10 @@ versions:
     # a short description of the version (optional)
     description: Yocto 3.1
 
+    # The CONF_VERSION to set in the config. Also, controls some compatibility
```

Comment:
Any point in using the value of the YP release numbers here rather than inventing your own monotonic counter?

Reply:
Ya, maybe it would be better to use our own..... I think we could use that to handle the override syntax change also, since #6 doesn't actually work (`+=` can't be used because the default is weakly assigned _after_ it, so it destroys the default instead of appending to it

Reply:
Okay, so you prefer to keep your own sequence values (1, 2, 3, ...) as written here, rather than adopt whatever numeric values YP uses to indicate its releases?